scripts/utils.sh

```diff
@@ -129,6 +130,15 @@ function setNginxHostnames {
 
 }
 
+function exportScheme() {
+    if [ "$USE_HTTPS" == "true" ]; then
+        export SERVER_SCHEME="https"
```

Comment:
Is "scheme" the right term here?
Isn't this rather a protocol, or transfer protocol?

Reply:
Either "scheme" or "protocol" works for me.

See the link below;
https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#:~:text=The%20first%20part%20of%20the%20URL%20is%20the%20scheme%2C